examples/presigned_post_policy.py

```diff
@@ -33,10 +33,21 @@
 
 form_data = client.presigned_post_policy(policy)
 
+# Decode from bytes so POST will work by having the policy as policy=eyJleHBpcm
+# If policy is like "policy=b'eyJleHBpcm..." Then this will fail.
+if isinstance(form_data['policy'], (bytes, bytearray)):
+    form_data['policy'] = form_data['policy'].decode("utf-8")
```

Comment:
value of `form_data["policy"]` should contain string. This bug needs to fixed at https://github.com/minio/minio-py/blob/master/minio/datatypes.py#L777 like `policy = base64.b64encode(json.dumps(policy).encode()).decode("utf-8")`

Reply:
Oh wow, you are right, this is much better than just converting in the example!!!.

Thanks Bala, really appreciate it, let fix the bug in a separate PR then.

Reply:
Let's fix it in this PR.